### ISSUE TITLE
Fix jupyterlab-myst (tornado), or remove from build

### ISSUE THREAD

**Issue by mattcen** [jupyterlite/jupyterlite#1122]
## Description

`jupyterlab-myst` is included by 3217ae2, but when trying to install it, I get an error about failure to find a suitable build of `tornado`:

## Reproduce

```python
%pip install jupyterlab-myst
```
```
ValueError: Can't find a pure Python 3 wheel for 'tornado>=6.2.0'.
See: https://pyodide.org/en/stable/usage/faq.html#micropip-can-t-find-a-pure-python-wheel
You can use `micropip.install(..., keep_going=True)`to get a list of all packages with missing wheels.
```

## Expected behavior

`jupyterlab-myst` installs successfully

## Context

- JupyterLite version: Tested on https://jupyter.org/try-jupyter/lab/

**Comment by jtpio** [comment]
Thanks @mattcen for reporting.

`jupyterlab-myst` used to be working with JupyterLite fine at some point: https://github.com/jupyterlite/jupyterlite/pull/1021

This issue likely comes from the dependency on `jupyter_server`: https://github.com/executablebooks/jupyterlab-myst/blob/23d2aba01353f02843cb298dd0361ff1b45b01db/pyproject.toml#L25

Which was added in https://github.com/executablebooks/jupyterlab-myst/pull/155

In the meantime we might want to pin on an earlier version, for example `jupyterlab-myst<2`.

**Comment by jtpio** [comment]
Pinning on `jupyterlab-myst<2` should help for now, and is also recommended until there is a new release of JupyterLite using the JupyterLab 4 packages (#1019):

![image](https://github.com/jupyterlite/jupyterlite/assets/591645/d718e44c-ba9b-4469-9abd-21c5d967b181)

**Comment by agoose77** [comment]
Fixed by `jupyterlab-myst>=2.0.2`

**Comment by jtpio** [comment]
Nice!

Maybe we can bump the extension here:

https://github.com/jupyterlite/jupyterlite/blob/fa5f44ff3f0eec2c75cada62fa659f13c9f26ab0/examples/jupyter_lite_config.json#L19
